src/Controls/src/Core/Layout/FlexLayout.cs

```diff
@@ -471,19 +471,46 @@ void InitItemProperties(IView view, Flex.Item item)
 			}
 		}
 
+#pragma warning disable RS0016 // Add public types and members to the declared API
```

Comment:
isn't this just going around the reason we added the analizer? shouldn't this fix go to main/net7 ?

Reply:
maybe a `TODO: Remove on NET7`